homeassistant/components/mqtt/abbreviations.py

```diff
@@ -126,6 +126,7 @@
     "pl_arm_nite": "payload_arm_night",
     "pl_arm_vacation": "payload_arm_vacation",
     "pl_arm_custom_b": "payload_arm_custom_bypass",
+    "pl_aut_zn": "payload_auto_zone",
```

Comment:
I had a discussion with @emontnemery about this PR. May we we do not need a new option.
We could use reuse `CONF_PAYLOAD_RESET` (`pl_rst`: `payload_reset`). We we want to allow is to reset the last location set.

Reply:
Ah, that may be a better solution indeed, yeah.

I see the rest of the changes cascading into this change, I'll apply it and change documentation appropriately, thank you for the feedback 👍